docs/release_notes/v1.11.5.md

```diff
@@ -19,3 +49,58 @@ An issue was detected in the internal implementation of service invocation when
 ### Solution
 
 We fixed the issue in Dapr 1.12.0 and 1.11.5.
+
+## Fixed: Fix invoke remote stream from closing stream before EOF
+
+### Problem
+
+If an invoke stream is using resiliency, then service invocation will return with an EOF error rather than successfully returning the result.
```

Comment:
```suggestion
When performing service invocation using resiliency, the service invocation can end with an EOF error rather than successfully returning the result.
```